src/types/undo-redo/moves/addRemoveDevice.ts

```diff
@@ -48,24 +50,69 @@ export class AddDeviceMove extends AddRemoveDeviceMove {
 // Check if the viewgraph is the best place to load the move into the manager
 export class RemoveDeviceMove extends AddRemoveDeviceMove {
   type: TypeMove = TypeMove.RemoveDevice;
+  data: CreateDevice; // Data of the removed device
+  connections: DeviceId[];
+  private storedRoutingTables: Map<DeviceId, RoutingTableEntry[]>;
+
+  constructor(
+    data: CreateDevice,
+    connections: DeviceId[],
+    viewgraph: ViewGraph, // Pasamos la vista para obtener las tablas de enrutamiento
```

Comment:
We shouldn't need to do this anymore. I previously refactored `CreateDevice` to store the node data from the datagraph, so if the data is in the datagraph, `CreateDevice` has it.